components/actions/ActionHighlightCard.tsx

```diff
@@ -57,8 +57,12 @@ const StyledCardTitle = styled(CardTitle)`
   font-size: ${(props) => props.theme.fontSizeMd};
   color: ${(props) => props.theme.neutralDark};
   text-align: left;
-  hyphens: manual;
+  hyphens: auto;
   margin-bottom: 0;
+
+  :lang(fi) & {
```

Comment:
Does this need to have an `&` to apply the `lang` pseudoclass to this element rather than the child elements? (same question for the other uses of this class)

```suggestion
  &:lang(fi) & {
```

Reply:
@lilia1891 Thanks for the clarification! In that case, this looks great